SofaKernel/modules/SofaCore/src/sofa/core/collision/BroadPhaseDetection.h

```diff
@@ -41,9 +40,11 @@ class BroadPhaseDetection : virtual public Detection
 {
 public:
     SOFA_ABSTRACT_CLASS(BroadPhaseDetection, Detection);
+
+    using CollisionModelPair = std::pair<core::CollisionModel*, core::CollisionModel*>;
 protected:
     /// Destructor
-    ~BroadPhaseDetection() override { }
+    ~BroadPhaseDetection() override = default;
 public:
```

Comment:
Basically there is no difference (there are some in some specific cases, but not here), but I like this notation because it is explicit and easy to read.